src/services/dbtTestService.ts

```diff
@@ -145,7 +145,9 @@ export class DbtTestService {
 
     const column =
       model.columns &&
-      model.columns.find((yamlColumn: any) => yamlColumn.name === column_name);
+      model.columns.find((yamlColumn: any) =>
+        isColumnNameEqual(yamlColumn.name, column_name),
+      );
```

Comment:
We won't need casing here?

Reply:
`column_name` is not from db, it is from metadata of test node, so it will match the case of schema.yml

Reply:
Variable name can be improved, please make it clear what is the source of the column_name. for example columnNameFromManifestTestData. Longer but will help if we need to look at this again in a few months.

Reply:
updated